### ISSUE TITLE
How much training time does it usually take?

### ISSUE THREAD

**Issue by ArmageddonKnight** [samsungsds-rnd/deepspeech.mxnet#21]
Hi,

Thank you for developing this very nice benchmark for the speech recognition application. However, could you please tell me how much time does it usually take for the training to converge (I am training on a single machine with a 1080 Ti GPU card)?

Currently I am on the 24k training iteration of the first epoch (which takes around 3 days to reach) and I am still getting prediction outputs that are very different compared with the label.

```
[    INFO][2019/06/13 12:20:35.564] label: in mail their horses clad yet fleet and strong prauncing their riders bore the flower and choice of many provinces from bound to bound from arachosia from candaor east 
[    INFO][2019/06/13 12:20:35.564] pred : th th th th th th  th the the th th    th th  th the th the the  th th th th  e  , cer: 0.711538 (distance: 74/ label length: 104)
[    INFO][2019/06/13 12:20:35.564] Epoch[0] Batch[23999] SAVE CHECKPOINT
```

Thanks in advance.

**Comment by Soonhwan-Kwon** [comment]
When I tested for benchmark, I tested on 3 1080 Ti GPU cards and it took almost 30 days. I think it can be dramatically reduced when using FusedRNNCell or faster implementations but I'm not sure about the accuracy.

**Comment by ArmageddonKnight** [comment]
@Soonhwan-Kwon That is a long time. Previously when I did benchmarking on NMT models, I noticed that `FusedRNNCell` can potentially speed up the training by 2 to 3 times. The issue is that since cuDNN adopts a completely different interface, programming it might be quite challenging (as is indicated [here](https://github.com/awslabs/sockeye/blob/master/CHANGELOG.md#1130)).

Thank you so much for your input, and once again, for open-sourcing this nice benchmark on the DeepSpeech model.
